schemas/dwh.sql

```diff
@@ -134,16 +133,14 @@ ON CONFLICT DO NOTHING;
 CREATE TABLE IF NOT EXISTS d_character
 (
     id        SERIAL PRIMARY KEY,
-    id_gender INT REFERENCES d_gender (id),
     name      TEXT NOT NULL UNIQUE
 );
 
 CREATE TABLE IF NOT EXISTS d_people
 (
     id        SERIAL PRIMARY KEY,
     id_gender INT REFERENCES d_gender (id),
-    name      TEXT NOT NULL,
-    id_tmdb   TEXT NOT NULL UNIQUE
+    name      TEXT NOT NULL
```

Comment:
We don't have any ways to get their names, but we can use tmdb ids for this as names:
id_tmdb   INT NOT NULL UNIQUE